# 2D position and orientation of the bounding box center
yolov8/Pose2D center

# total size of the bounding box, in pixels, surrounding the object's center
yolov8/Vector2 size